_includes/plugins/katello/sidebar_3.11.html

```diff
@@ -0,0 +1,180 @@
+<ul class="header nav nav-list bs-docs-sidenav">
+  <li class="item">
+    <a>1. Installation</a>
+  </li>
+
+  <ul class="sub-header-static nav nav-list bs-docs-sidenav">
+    <li class="item">
+      <a href="/plugins/katello/3.11/installation/index.html">1.1 Katello</a>
+    </li>
+    <li class="item">
+      <a href="/plugins/katello/3.11/installation/smart_proxy.html">1.2 Smart proxy</a>
+    </li>
+    <li class="item">
+      <a href="/plugins/katello/3.11/installation/clients.html">1.3 Clients</a>
+    </li>
+  </ul>
+</ul>
+
+<ul class="header nav nav-list bs-docs-sidenav">
+  <li class="item">
+    <a>2. Upgrade</a>
+  </li>
+
+  <ul class="sub-header-static nav nav-list bs-docs-sidenav">
+    <li class="item">
+      <a href="/plugins/katello/3.11/upgrade/index.html">2.1 Katello</a>
+    </li>
+    <li class="item">
+      <a href="/plugins/katello/3.11/upgrade/smart_proxy.html">2.2 Smart proxy</a>
+    </li>
+    <li class="item">
+      <a href="/plugins/katello/3.11/upgrade/clients.html">2.3 Clients</a>
+    </li>
+    {% if page.version == 'nightly' or page.version == 3.2 %}
```

Comment:
I think we can drop the puppet upgrade that was needed in 3.2 now? :)

Reply:
(doesn't have to be in this PR, we'd be updating other versioned pages too then anyways)

Reply:
removed this check for 3.11 and nightly & removed upgrades/puppet.md for > 3.2